grakn-engine/src/main/java/ai/grakn/engine/postprocessing/UpdatingInstanceCountTask.java

```diff
@@ -73,10 +81,9 @@ private void updateCountsOnKeySpace(String keyspace){
                 graknGraph.admin().updateTypeCounts(jobs);
                 graknGraph.admin().commitNoLogs();
             } catch (Throwable e) {
-                LOG.error("Unable to updating instance counts of graph [" + keyspace + "]", e);
+                LOG.warn("Unable to updating instance counts of graph [" + keyspace + "]", e);
                 if(retry > 10){
-                    LOG.error("Failed 10 times in a row to update the counts of the types on graph [" + keyspace + "] giving up");
-                    jobs.forEach((key, value) -> cache.addJobInstanceCount(keyspace, key, value));
+                    throw new RuntimeException("Failed 10 times in a row to update the counts of the types on graph [" + keyspace + "] giving up");
```

Comment:
Now it will not resubmit them to the cache. A task cannot re-schedule or re-submit itself (at the moment), so when committing fails 10 times, those counts are lost.